### ISSUE TITLE
Uhyve: Implement `chdir`

### ISSUE THREAD

**Issue by n0toose** [hermit-os/kernel#1483]
Related to https://github.com/hermit-os/uhyve/issues/815.

**Comment by jounathaen** [comment]
@n0toose is this still relevant? If so, could you add a brief description the issue is?

**Comment by jounathaen** [comment]
I don't think this is an issue. Hermit should access all Uhyve files with a full path. If it isn't, then we should create a specific issue for that.
